environment.yml

```diff
@@ -4,7 +4,7 @@ channels:
   - conda-forge
 
 dependencies:
-  - cftime>=1.3.0  # years<999 get a 0 instead of empty space
+  - cftime=1.2.1  # iris3 needs <=1.2.1; >=1.3.0 - years<999 get a 0 instead of empty space
   - compilers
   - esmpy
   - iris>=2.2.1
```

Comment:
```suggestion
  - iris>=3
```

Reply:
right! this is an important one, good spot!